packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -0,0 +1,130 @@
+const _partnerId = process.env.KALTURA_PARTNER_ID || 243342;
+const _uiConfId = process.env.KALTURA_UICONF_ID || 12905712;
```

Comment:
Add JSDocs here. Should make some mention that these values are for production for IBM.com.

Reply:
Sure